schemas/com.mailgun/message_bounced/jsonschema/1-0-0

```diff
@@ -0,0 +1,97 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a Mailgun message bounced event",
+  "self": {
+    "vendor": "com.mailgun",
+    "name": "message_bounced",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "attachment-count": {
+      "description": "Number of attachments in attachment-x (where x is a number) fields.",
+      "type": "integer",
+      "maximum": 32767
+    },
+    "body-plain": {
```

Comment:
I'd still go with removing it from payload. If I understand it right - this will repeat multiple times (newsletter) and has no real value. I can be wrong.

Reply:
I thought the same. As you say this will be almost the same in all messages and may cause other problems (e.g. kinessis choking on it).